content/models.py

```diff
@@ -27,3 +27,17 @@ class OurStoryTitle(models.Model):
 
     def __str__(self):
         return self.title
+
+class ChildrensProgram(models.Model):
+	pagetitle = models.CharField(blank=True, null=True, max_length=100)
+	img = models.ImageField(upload_to='education',blank=True, null=True)
```

Comment:
also looks like the template errors if you dont input an img. you should probably make this required as well?